externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/operation/ExecuteStatement.scala

```diff
@@ -69,13 +70,18 @@ class ExecuteStatement(
     OperationLog.removeCurrentOperationLog()
   }
 
+  override val redactedStatement: String =
+    redact(spark.sessionState.conf.stringRedactionPattern, statement)
+
   private def executeStatement(): Unit = withLocalProperties {
     try {
       setState(OperationState.RUNNING)
       info(diagnostics)
       Thread.currentThread().setContextClassLoader(spark.sharedState.jarClassLoader)
       // TODO: Make it configurable
       spark.sparkContext.addSparkListener(operationListener)
+      spark.sparkContext.setJobDescription(
+        redact(spark.sessionState.conf.stringRedactionPattern, statement))
```

Comment:
Can we make redacted statment at:

https://github.com/apache/incubator-kyuubi/blob/cb8721fef130771db330c0dd51844ebb13bca7b3/externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/operation/SparkOperation.scala#L98-L100

And we can also define the `redactedStatement` at `SparkOperation`.

Reply:
+1 to set there

Reply:
let me try

Reply:
move codes to `SparkOperation`